src/backend/commands/tablecmds.c

```diff
@@ -927,7 +927,8 @@ DefineRelation(CreateStmt *stmt, char relkind, Oid ownerId,
 	 */
 	if (intoPolicy)
 	{
-		Assert(!stmt->inhRelations);
+		/* storage_am plguin rewrite intoPolicy, so just comments it.*/
+		//Assert(!stmt->inhRelations);
```

Comment:
If we don't need Assert here, please remove it.

Reply:
done